engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -186,6 +189,21 @@ class EngineJobExecutionActor(replyTo: ActorRef,
       stay using data.copy(hashes = Option(Failure(t)))
   }
 
+  when(InvalidatingCacheEntry) {
```

Comment:
ToL: All this in the EJEA is making me more and more convinced a separate `DoACacheHit` is needed. Not in this PR though :)